### DISCUSSION TITLE
Build complete. Now to figure out how to use it!

### DISCUSSION THREAD

**Discussion by jamietaylor13** [FundamentalFrequency/.github#81] [Show and tell]
Thought I would share my build here and ask if anyone has a link to a manual. I've watched the videos but wondered if anyone had compiled the full list of options and features.

The keys are a bit uneven due to my bad soldering job but will rectify this very shortly. Link to my thingiverse files if anyone is interested in having these keys and the keyplate to go with case designed by fofinho666 (nice job!). These can be printed on a 200x200mm printbed suitable for home printing.
Keys and Plate: https://www.thingiverse.com/thing:5996834
Case: https://www.thingiverse.com/thing:5833535
![featured_preview_24cf3c84-ba85-427c-96da-a32042806cee](https://github.com/FundamentalFrequency/.github/assets/3817043/20403c62-a5d5-4acf-8c55-a0db7be2e3b1)

**Comment by iamdey** [comment]
Is it here where we all show our builds ? :)

**Reply by iamdey** [reply]
Here is mine, (well already done since a while actually)

![lmn-3](https://github.com/user-attachments/assets/f2158722-1b72-41bb-97e4-07a7bbdbd236)

Also [I've detailed the build steps](https://www.bruit-contraire.net/lmn-3-daw) in a blog post in french from disassembling an old keyboard to cut the case with a laser.

**Comment by vidalsasun** [comment]
This is my setup

![lmn-3](https://github.com/user-attachments/assets/3768fad8-ba3f-45af-9d70-4df634bc4ae3)

i have used a 3.5 screen because i chaeper than hyperpixel 👍
